### PR TITLE
move templates to ISSUE_TEMPLATES

### LINKED ISSUE #3741: Improving multiple Issue Templates
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [.github](https://github.com/publiclab/plots2/tree/master/.github) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
Make a folder with name ISSUE_TEMPLATE and put all our issue templates inside this folder.
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by sandy8111112004**
Fixes #3741 

1. move first-timers-issue-template.md, ISSUE_TEMPLATE.md, PULL_REQUEST_TEMPLATE.md 
    to ISSUE_TEMPLATES folder

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, welcome, and thank you for your contribution! @DivyeshPuri, were you able to help review this? Also, will GitHub recognize the templates when placed in a sub-folder? Perhaps there was some documentation you were reading about where these can be placed? 

Thank you both!

**Comment by jywarren** [comment]
Oh awesome! Thanks a lot :-)

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (jywarren) [inline package-lock.json]: Hi, this is looking great, but would you mind removing the package-lock.json from your PR, as I think it's just a version tweak? https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Comment (sandy8111112004) [reply package-lock.json]: Hi, thanks for the feedback. I'll fix it soon.

**Pull request closed without merging**